src/CoreBundle/private/library/classes/TCMSTreeWidget/TCMSTreeWidget.class.php

```diff
@@ -420,7 +420,6 @@ public function GetHtmlHeadIncludes()
     {
         $aIncludes[] = '<script src="'.TGlobal::GetStaticURL(URL_CMS.'javascript/jquery/cookie/jquery.cookie.js').'" type="text/javascript"></script>';
         $aIncludes[] = '<script src="'.TGlobal::GetStaticURL(URL_CMS.'javascript/jquery/jsTree/jquery.jstree.js').'" type="text/javascript"></script>';
-
```

Comment:
Please revert this change (add the line again).

Reply:
ok